oauth2client/django_util/__init__.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2015 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Utilities for the Django web framework
+
+Provides Django views and helpers the make using the OAuth2 web server
+flow easier. It includes a ``required`` decorator to automatically ensure
+that user credentials are available.
+
+
+Configuration
+=============
+
+First, add the helper to your INSTALLED_APPS:
+
+INSTALLED_APPS = (
+    # other apps
+    "oauth2client.django_util"
+)
+
+To configure, you'll need a set of OAuth2 web application credentials from
+`Google Developer's Console <https://console.developers.google.com/project/_/\
+apiui/credential>`__.
+
+.. code-block:: python
+   :caption: settings.py
+   :name: settings-file
+
+   GOOGLE_OAUTH2_CLIENT_SECRETS_JSON=client-secret.json file
+
+Or,
+
+.. code-block:: python
+   :caption: settings.py
+
+
+   GOOGLE_OAUTH2_CLIENT_ID=client-id-field
+   GOOGLE_OAUTH2_CLIENT_SECRET=client-secret-field
+
+By default, the default scopes for the required decorator only contains the
+(email) scopes. You can change that default in the settings.
+
+GOOGLE_OAUTH2_SCOPES = ('https://www.googleapis.com/auth/calendar',)
+
+
+then in your urls.py
+
+.. code-block:: python
+   :caption: urls.py
+
+   from oauth2client.django_util.site import urls as oauth2_urls
+
+   urlpatterns += [url(r'^oauth2/', include(oauth2_urls))]
+
+Finally, on any of your views add the decorator
```

Comment:
Change to "To require OAuth2 credentials for a view, use the `required` decorator:".